conftest.py

```diff
@@ -206,7 +205,10 @@ def hashing(request):
             # As such, if we're comparing and the file of saved hashes isn't
             # found, we just skip the test. We do the skip before the test
```

Comment:
If the test fails now this comment should be updated.

Reply:
MMMM Nothing sounds better than _mystery_ 🦇 🌔 🕯️ 🙂 

I'm good with this removal if PR 3 is very clear about what's going on!